app/proxy_pages.rb

```diff
@@ -0,0 +1,110 @@
+class ProxyPages
+  def self.resources
+    publishing_api_docs +
+      govuk_schema_names +
+      app_docs +
+      app_docs_json +
+      document_types +
+      supertypes
+  end
+
+  def self.publishing_api_docs
+    PublishingApiDocs.pages.map do |page|
+      {
+        url: "/apis/publishing-api/#{page.filename}.html",
+        template: "templates/publishing_api_template.html",
+        frontmatter: {
```

Comment:
I'm unsure about the naming `frontmatter` here as I was a bit unsure what this was, perhaps `options` is a better name here...

Reply:
Agreed, it's not very intuitive, except that it is Googlable ([it's what Middleman uses](https://middlemanapp.com/basics/frontmatter/) to refer to the page-level attributes, so seems reasonable to use the same terminology). I think it makes it clearer that it's a Middleman-specific thing, rather than some app-specific thing we've come up with ourselves. Happy to rename if you think we should though.

Reply:
Hmm, I was under the assumption that `locals` was the frontmatter (i.e. they get used when rendering the page), and the third argument to the function was just an `options`. I don't think it matters too much though... I'm happy to leave it as it is.